### ISSUE TITLE
Object list for includes

### ISSUE THREAD

**Issue by MDagni** [fidley/ADT-Classic-Outline-Backend#12]
When an include is opened in ADT, classic outline shows only the objects of that include. You can make it show the main program's objects if you add this to ZCL_ADTCO_TREE_CREATOR=>GET_OBJECT_NAME line 13:

`      WHEN 'REPS'.
        object_name+40 = original_object_name.`

Example program DEMO_DYNPRO_MODIFY_SCREEN, include DEMO_DYNPRO_MODIFY_SCREEN_SEL.

Before the change:
![image](https://user-images.githubusercontent.com/33897441/157053807-b418785b-afaa-4f9f-89ed-64039f5789d2.png)

After the change:
![image](https://user-images.githubusercontent.com/33897441/157053920-7d3e388f-d916-4ef6-ab7d-2a7493849cc3.png)

I would like to push this change myself, but abapgit shows so many differences when I try to push and I don't want to look like I've done more than I actually did...

**Comment by fidley** [comment]
I was promoting it few times already :) Including the demo on the ABAPConf :)
I will try to push this today ( and test :)

**Comment by fidley** [comment]
Thanks a lot

**Comment by MDagni** [comment]
Ah, I missed all the good stuff in ABAPConf :( The recordings are still not available..

**Comment by fidley** [comment]
In fact it works if the Main program is assinged to include:

![image](https://user-images.githubusercontent.com/7912195/157078985-7cc8ac12-ee09-4c30-8f5d-6805ac78d2bd.png)
In other cases seems not ;)

**Comment by MDagni** [comment]
You're right, it only works if the include is used by only one program. This check is done in FM WB_TREE_CONVERT_OBJECTNAME line 132.

It might be better if you can read that Main Program selection on the properties view and send it instead of include name for tree generation.

**Comment by fidley** [comment]
I could see some options here:

1. When you open include from double click on the Classic Outline -> take the tree/main program from the currently open tree
2. When you open include manually, then search for the main program in DB (if there is such thing), if it's empty or not exists, select first one? Or leave as currently only with the include info?

**Comment by MDagni** [comment]
1st option would be useful. But generally I tend to open includes by ctrl+clicking on a form or method name and ADT finds the include. Would be nice if you could catch that.

For 2nd option; if there is only one program, it automatically chooses that one in current situation. If there are multiple, it would be confusing if you choose first one because most likely it won't be the one I'm working on. So leaving it with include info is better.

**Comment by fidley** [comment]
So far I don't know how to get the info from the PropertiesView in Eclipse, that's why it is a bit tricky for the Main Program. I will think about it.

**Comment by fidley** [comment]
d010inc table have the master program :)

**Comment by fidley** [comment]
Please update Frontend & Backend components. Should work with the latest changes correctly.

**Comment by MDagni** [comment]
Looks really good :) If the include is used in more than one program, it shows the first one it finds, but that's a lot better than nothing.
